kolibri_explore_plugin/views.py

```diff
@@ -183,11 +183,20 @@ class EndlessLearningCollection(View):
         },
     }
 
-    def check_collection_availability(self):
+    def update_collection_json(self):
         free_space_gb = get_free_space() / 1024**3
-        for collections in self.grade_collections.values():
-            for v in collections.values():
+        for grade, collections in self.grade_collections.items():
+            for collection, v in collections.items():
+                # check collection availability
                 v["available"] = v["size"] < free_space_gb
```

Comment:
What is "v" in this context? Please use a meaningful variable name.

Reply:
I've added a new commit using better variable names.